### ISSUE TITLE
Crushes if there is no value added instatly

### ISSUE THREAD

**Issue by stsdc** [lcallarec/live-chart#16]
So if there is no value in a serie and I'm adding chart to the box It throws multiple `** (com.github.stsdc.monitor:22174): CRITICAL **: 00:02:25.398: gee_collection_get_size: assertion 'self != NULL' failed`.

```vala
    public Chart (int series_quantity) {
        for (int i = 0; i < series_quantity; i++) {
            var renderer = new LiveChart.SmoothLineArea (new LiveChart.Values(1000));
            var serie = new LiveChart.Serie (("Serie %d").printf(i), renderer);
            serie.set_main_color ({ 0.35 + i/20, 0.8, 0.1, 1.0});
            live_chart.add_serie (serie);

            //  live_chart.add_value (serie, 0);
        }
        add (live_chart);
    }
```

**Comment by stsdc** [comment]
So there also will be an error if I'll do this in another chart so I have to make it with : `live_chart.add_value (serie, 3);` because if there is **`0`** it throws an error:
`ERROR:arraylist.c:1197:gee_array_list_real_get: assertion failed: (index >= 0)`

The Monitor code: https://github.com/stsdc/monitor/blob/ea5a70f1c83e348d66daeb7af79b72f6a3026cb9/src/Widgets/Chart/NetworkChart.vala#L36-L49

**Comment by lcallarec** [comment]
Thanks for the reports ; and good catch !

Both issues are fixed in [release 1.6.0](https://github.com/lcallarec/live-chart/releases/tag/1.6.0). 

Please tell me if it really fixes your own use cases.

**Comment by stsdc** [comment]
Works perfectly.
